source/nanoFramework.Json/DebugHelper.cs

```diff
@@ -13,7 +16,7 @@ internal static class DebugHelper
         // Used this a lot in the effort to get Array serialization & deserialization working
         public static void DisplayDebug(string displayString)
         {
-#if DEBUG
+#if NANOFRAMEWORK_DISPLAY_DEBUG
```

Comment:
nanoFramework will automatically show the `Console.WriteLine` when #if DEBUG is used and the lib is in debug mode, there is no reason for NANOFRAMEWORK_DISPLAY_DEBUG here. Please revert. (note, it is now preferable to use `Debug.WriteLine` which removes the need for this function entirely)

Reply:
I thought about changing everything to Debug.WriteLine, but there's a lot of output.
Don't most people working in Debug mode?
If so, the serialize/deserialize output would be annoying. 
I thought NANOFRAMEWORK_DISPLAY_DEBUG would be a convenient way to turn on the lib output if we ever need to work on it again.

Reply:
Not sure how Debug works when referencing published nugets… @josesimoes will Debug be output when published and referenced?